### ISSUE TITLE
Mocking medical record data

### ISSUE THREAD

**Issue by IanMacDougall** [VizerISD/personalhealthcarewallet#53]
Users want to see different data displayed for different assets. Currently, any and all assets will display the same data into the visualizations (medical record access history and most frequent accessors). Once the JSON data from #50 is merged into main, we can begin mocking the data displayed for each of the assets. 

Clicking a medical record on the profile page will take you to the respective asset page. There, we need to change the following content/data:
1. Data for the Most Frequent Accessors viz
2. Data for the Medical Record Access History viz
3. Description information about the asset
4. Syncing the appropriate data file to the asset download button
5. (Potentially) different doctors with access for each asset. 

Note that the above list can be split into individual issues if different team members want to take different parts. 

For clarity about how to make these changes, refer to the code base and review the current implementation on how mocked data is transposed into base "Medical Record 2" asset. If there are questions about implementation feel free to ping @IanMacDougall.
